src/jsonrpc/miner_jsonrpc_ledger.erl

```diff
@@ -10,6 +10,8 @@
 %% jsonrpc_handler
 %%
 
+-type json_object() :: #{ binary() => any() }.
```

Comment:
I would not recommend doing this..there is 1. already a type spec for a json object out of jsx I believe
and 2. you can actually use atoms for keys with the used library.. Other jsonrpc endpoints do this

Reply:
This module has already established the convention of using `binary()` for keys. I didn't want to rock the boat. I can try to find the jsx type...

Reply:
the module is tiny.. you should have gone with the middle ground if you didn't want to rock the boat

Reply:
I think `json_object() :: #{ binary() => any() }.` is the middle ground, no?